notify.lua

```diff
@@ -71,14 +71,17 @@ http = require("socket.http")
 http.TIMEOUT = 3
 http.USERAGENT = "mpv-notify/0.1"
 
+posix = require("posix")
+
 local CACHE_DIR = os.getenv("XDG_CACHE_HOME")
 CACHE_DIR = CACHE_DIR or os.getenv("HOME").."/.cache"
 CACHE_DIR = CACHE_DIR.."/mpv/coverart"
 print_debug("making " .. CACHE_DIR)
 os.execute("mkdir -p -- " .. string.shellescape(CACHE_DIR))
 
 function tmpname()
-	return "/tmp/mpv-coverart." .. math.random(0,0xffffff)
+	fd, fname = posix.mkstemp(CACHE_DIR .. "/rescale.XXXXXX")
```

Comment:
`fd` is just an integer I think. I don't know Lua that well though. It might need an `os.close` or something.